catalog/common/catalog-common-geo-formatter/src/main/java/ddf/geo/formatter/CompositeGeometry.java

```diff
@@ -159,7 +159,7 @@ private static double getDouble(Object object) {
    * @return an array of {@link Coordinate} objects, if input is <code>null</code> or empty, it will
    *     return an empty array
    */
-  protected static Coordinate[] getCoordinates(List<List<String>> primitiveCoordinatesList) {
```

Comment:
✏️ Why was this changed from `List<List<String>>` to `List<List>`? Can we get away with specifying the `List` type?

Reply:
i think so. This method can accept any Object, the Object will be converted to a double.